python/ray/_private/services.py

```diff
@@ -191,7 +191,10 @@ def get_address_info_from_redis_helper(redis_address,
             break
     if relevant_client is None:
         raise RuntimeError(
-            "Redis has started but no raylets have registered yet.")
+            "Connected to Redis at {} and found raylets at {} but none of these"
```

Comment:
Just a minor comment, but can you use fstring instead?

Reply:
Also, it'll be nice to add some action items here. Do you have any good suggestion?

Reply:
@rkooo567 I added an action item of a sort (https://github.com/ray-project/ray/pull/11957/commits/af557aa6902f94f062d187507b1ddbd664951853), not sure if that's what you have in mind. Really, I kind of want to change the ad-hoc
```
        if (client_node_ip_address == node_ip_address
                or (client_node_ip_address == "127.0.0.1"
                    and redis_ip_address == get_node_ip_address())):
```
business to a separate alias-resolution function. But that's a a more radical change and I don't fully understand everything `get_address_info_from_redis_helper` is doing. Plus https://github.com/ray-project/ray/pull/11944 is in the process of changing it and I really don't understand how that works (I thought this would only ever be called with `redis_address` as the the address of the head node, but I guess not always).